docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -4,18 +4,15 @@ skinparam arrowThickness 1.1
 skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
-AddressBook *-right-> "1" UniquePersonList
-AddressBook *-right-> "1" UniqueTagList
-UniqueTagList -[hidden]down- UniquePersonList
-UniqueTagList -[hidden]down- UniquePersonList
+ModuleTracker *-right-> "1" UniqueModuleList
+ModuleTracker *-right-> "1" UniqueTagList
+UniqueTagList -[hidden]down- UniqueModuleList
+UniqueTagList -[hidden]down- UniqueModuleList
```

Comment:
What is this code doing? Sorry I havent had the chance to look at the DG code yet.

Reply:
![image](https://user-images.githubusercontent.com/122522367/230717385-88b0894b-3cb5-49ca-b364-dc09c7ef3654.png)
For example, line 7 creates an "arrow" from `ModuleTracker` to `UniqueModuleList` to the "right", with indication that there can only be "1" "UniqueModuleList".